src/components/SearchRecords.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Message from './message';
+import { Typography } from '@material-ui/core';
+import { makeStyles } from '@material-ui/styles';
+
+const useStyles = makeStyles({
+  searchStyle: {
+    fontStyle: 'oblique',
+    color: 'red',
+    fontSize: '30px',
+  },
+});
```

Comment:
This is my stylistic preference, but I prefer putting the styles at the bottom of the page instead of the top. That way I don't have to scroll past a bunch of stuff before getting to the meat of the code.

Practically all the code I've seen puts the styles at the top, so I'm probably unusual. Part of the reason for that is there is a default eslint rule that says you have to declare things before you use them. JavaScript and Typescript don't require that. I think it's a stupid rule. I honestly can't think of a single good reason anyone would want to enforce that, so I've turned it off. Code can be written so much more clearly if you sometimes move things down the file, out of the way.

Reply:
I really appreciate this, I always found out annoying too and I'm glad to get that pushed out of the way early. If there are any rules or habits you want me to follow, please let me know :-)